vmdb/app/views/layouts/quadicon/_ems_cluster.html.haml

```diff
@@ -0,0 +1,20 @@
+- width = size == 150 ? 54 : 35
+- height = width
```

Comment:
this is calculated over and over in several of these templates...
can we have less variable, less places with the calculation and use the helper mentioned above for this?